sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CheckNameAvailabilityContent.Serialization.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.CosmosDB.Models
+{
+    public partial class CheckNameAvailabilityContent : IUtf8JsonSerializable
```

Comment:
`CheckNameAvailabilityContent` is too generic, consider renaming to `CheckCosmosDBNameAvailabilityContent`
```yaml
CheckNameAvailabilityRequest: CheckCosmosDBNameAvailabilityContent
```